package.json

```diff
@@ -44,6 +44,7 @@
     "docsify-cli": "^4.4.3",
     "fastify": "^4.17.0",
     "graphql-ws": "^5.11.2",
+    "mercurius-validation": "^4.0.0",
```

Comment:
why a dependency on this? if you want to write a test that check that the integration with mercurius-validation works fine, that test should be written in that repo, not this one